lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -2511,6 +2511,8 @@ static void collectMinimalIndicesForFunctionCall(
       paramIndices.push_back(currentParamIdx);
     ++currentParamIdx;
   }
+  assert((currentParamIdx > 0 || !paramIndices.empty())
```

Comment:
Just wanted to verify we still want this assert, or maybe change the string as there is a `"Parameter indices cannot be empty"` assert inside `visitApplyInst`

Reply:
I just dropped it. Thanks for the catch!